submissions/Test123423235/CarRacingGame/IRenderObject.java

```diff
@@ -0,0 +1,5 @@
+package CarRacingGame;
+
+public interface IRenderObject {
+    public abstract void render();
```

Comment:
`public abstract` is the default in an interface and is therefore not needed. But it is not wrong.